maps/away/ships/tirakqi_smuggler/tirakqi_smuggler.dm

```diff
@@ -5,7 +5,7 @@
 	prefix = "ships/tirakqi_smuggler/"
 	suffixes = list("tirakqi_smuggler.dmm")
 
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
+	sectors = list(ALL_BADLAND_SECTORS)
```

Comment:
this removes it from corporate sectors

set this to `sectors = list(ALL_TAU_CETI_SECTORS, ALL_BADLAND_SECTORS)`